index.css

```diff
@@ -322,8 +322,8 @@ ul {
 
 .content-tags {
   border-radius: 4px;
-  margin-top: 12px;
-  margin-bottom: 10px;
+  margin-top: 8px;
+  margin-bottom: 5px;
```

Comment:
please lets use the shorthand property here instead

Reply:
Ok